src/components/layout/GradientLayout/GradientLayout.tsx

```diff
@@ -0,0 +1,31 @@
+import { Container, Grid, Stack, Image } from "@mantine/core";
+import classes from "./gradientLayout.module.css";
+import { PropsWithChildren } from "react";
+import Logo from "../../../assets/logo.svg";
+
+export const GradientLayout: React.FC<PropsWithChildren> = ({ children }) => {
+  return (
+    <div className={classes.container}>
```

Comment:
Would use a `Box` container instead of the `div`, and add `{...rest}` (type will be something like `PropsWithChildren & BoxProps`)